CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+This repository uses a shared copyright model. Each contributor maintains copyright over their contributions to this repository. Thus, the source of the repository is not not the copyright of one single author or contributor. Instead, it is the collective copyright of Pyspark Playground Contributors. Individual contributors are free to indicate their copyright in the commit message.
```

Comment:
```suggestion
This repository uses a shared copyright model. Each contributor maintains copyright over their contributions to this repository. Thus, the source of the repository is not the copyright of one single author or contributor. Instead, it is the collective copyright of PySpark Playground Contributors. Individual contributors are free to indicate their copyright in the commit message.
```

Reply:
@yulyt Thanks :) I noticed that `not not` typo and I fixed it in 
29d36a7. The `Pyspark` is okay for the name of the group because it's just the legalese. Please let me know if you feel otherwise.